src/main/java/gwt/material/design/client/ui/MaterialLink.java

```diff
@@ -98,6 +99,15 @@ public void setText(String text) {
 		generateLink();
 	}
 
+	public String getHref() {
+		return href;
+	}
+
+	public void setHref(String href) {
+		this.href = href;
+		generateLink();
```

Comment:
I was not really sure about this line... is the HTML being generated 3 times if I have an icon, a label and a href? It seems weird, doesn't it?